test/cypress/integration/entry-editor-extension.spec.js

```diff
@@ -0,0 +1,34 @@
+import { entry } from '../utils/paths'
+
+const post = {
+  id: '5mwUiJB2kThfAG9ZnRNuNQ',
+  title: 'My post with a custom entry editor',
+  body: ''
+}
+
+context('Entry editor extension', () => {
+  beforeEach(() => {
+    cy.setAuthTokenToLocalStorage()
+  })
+
+  it('opens first post and checks that entry editor extension is rendered', () => {
+    cy.visit(entry(post.id))
+
+    cy.getByText(post.title).should('exist')
+
+    // eslint-disable-next-line
+    cy.wait(3000)
```

Comment:
can we introduce helper methods for waiting (`waitForField`, `waitForPage`...) so we:

- give them meaningful names
- centralize timeout values
- centralize eslint rule disablement